dask/dataframe/groupby.py

```diff
@@ -1784,3 +1784,26 @@ def aggregate(self, arg, split_every=None, split_out=1):
     @derived_from(pd.core.groupby.SeriesGroupBy)
     def agg(self, arg, split_every=None, split_out=1):
         return self.aggregate(arg, split_every=split_every, split_out=split_out)
+
+    #  @derived_from(pd.core.groupby.SeriesGroupBy)
+    def value_counts(self, split_every=None, split_out=1):
+        return self._aca_agg(
+            token="value_counts",
+            func=M.value_counts,
+            aggfunc=lambda s:
+                pd.Series(pd.concat({k: v.sum(level=1) for k, v in s})),
+            split_every=split_every,
+            split_out=split_out,
+        )
+
+    #  @derived_from(pd.core.groupby.SeriesGroupBy)
```

Comment:
If I un-comment this line, I get an exception, it looks like it's on the pytest import. Here's the traceback/error message I get when I run `pytest tests/test_groupby.py -k "groupby_value_counts or groupby_unique" --pdb`:

<details>

``` python
$ pytest tests/test_groupby.py -k "groupby_value_counts or groupby_unique" --pdb
================================= test session starts ==================================
platform darwin -- Python 3.7.3, pytest-5.1.2, py-1.8.0, pluggy-0.13.0
rootdir: /Users/scott/Developer/stsievert/dask, inifile: setup.cfg
collecting ...
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> traceback >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
/Users/scott/anaconda3/lib/python3.7/site-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
__init__.py:14: in <module>
    from .groupby import Aggregation
groupby.py:1617: in <module>
    class SeriesGroupBy(_GroupBy):
groupby.py:1695: in SeriesGroupBy
    def unique(self, split_every=None, split_out=1):
../utils.py:651: in wrapper
    original_klass, method, ua_args=ua_args, extra=extra
../utils.py:613: in _derived_from
    original_args = get_named_args(original_method)
../compatibility.py:120: in get_named_args
    s = inspect.signature(func)
/Users/scott/anaconda3/lib/python3.7/inspect.py:3083: in signature
    return Signature.from_callable(obj, follow_wrapped=follow_wrapped)
/Users/scott/anaconda3/lib/python3.7/inspect.py:2833: in from_callable
    follow_wrapper_chains=follow_wrapped)
/Users/scott/anaconda3/lib/python3.7/inspect.py:2208: in _signature_from_callable
    raise TypeError('{!r} is not a callable object'.format(obj))
E   TypeError: <property object at 0x120fbc868> is not a callable object
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> entering PDB >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

>>>>>>>>>>>>>>>>>>>>>> PDB post_mortem (IO-capturing turned off) >>>>>>>>>>>>>>>>>>>>>>>
> /Users/scott/anaconda3/lib/python3.7/inspect.py(2208)_signature_from_callable()
-> raise TypeError('{!r} is not a callable object'.format(obj))
(Pdb)
```

</details>

Reply:
Interesting.  If it's easy, can I ask you to investigate which object it's running into problems with?  Is it SeriesGroupBy itself?  I would guess not because it is, in fact, callable. 

```python
In [2]: import pandas as pd

In [3]: pd.core.groupby.SeriesGroupBy
Out[3]: pandas.core.groupby.generic.SeriesGroupBy

In [4]: callable(pd.core.groupby.SeriesGroupBy)
Out[4]: True
```

Navigating that traceback and figuring out how we got to that point would be helpful.